### ISSUE TITLE
Provide a way to disable url-loader for images

### ISSUE THREAD

**Issue by gajus** [vercel/next.js#17804]
# Feature request

## Is your feature request related to a problem? Please describe.

Default Next.js config adds all images under 8192 bytes to the bundle.

This can be problematic esp. if landing pages includes a lot of images that are loaded at different times, e.g.

In my specific case http://aboutsnack.com/ had floating SVGs and inlining them all to the bundle added extra 100KB.

Since technologies such as HTTP/2 enable an efficient way to load many smalls assets in parallel, it is detrimental to the overall user experience to inline all of these assets.

## Describe the solution you'd like

A simpler way to disable `url-loader` for images.

## Describe alternatives you've considered

I've patched webpack config:

```js
const imageRule = config.module.rules.find((rule) => {
  return String(rule.test) === String(/\.(jpe?g|png|svg|gif)$/i);
});

if (!imageRule) {
  throw new Error('Unexpected state');
}

const fallback = imageRule.oneOf[imageRule.oneOf.length - 1];

if (fallback.resourceQuery) {
  throw new Error('Unexpected state');
}

const urlLoader = fallback.use[0];

if (urlLoader.loader !== 'url-loader') {
  throw new Error('Unexpected state');
}

```

Just in case there is already a better solution, please share the answer to https://stackoverflow.com/a/64310347/368691.

**Comment by Timer** [comment]
> Default Next.js config adds all images under 8192 bytes to the bundle.

This is not the case, Next.js does not inline or support importing images. This is likely done by some custom configuration you've added to Next.js!

**Comment by gajus** [comment]
> This is not the case, Next.js does not inline or support importing images. This is likely done by some custom configuration you've added to Next.js!

Indeed it was. It was done by https://www.npmjs.com/package/next-optimized-images

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
